docs/website/docs/command-reference/describe-component.md

```diff
@@ -2,7 +2,7 @@
 title: odo describe component
 ---
 
-`odo describe component` command is useful for getting information about a component managed by `odo`. 
+`odo describe component` command is useful for getting information about a component. 
```

Comment:
This is still true, no?

Reply:
No, it isn't anymore since https://github.com/redhat-developer/odo/pull/6615